CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,58 @@
+- Sustained disruptions of community events and discussions;
+- Advocating for, or encouraging, any of the above behavior
+
+## Process for Moderation
+
+We encourage users to [work out issues directly](https://wiki.openstreetmap.org/wiki/Foundation/Local_Chapters/United_States/Code_of_Conduct_Committee/OSM_US_Code_of_Conduct#How_to_Handle_a_Complaint) with the people involved in lieu of a formal report.
+If you believe someone has violated the CoC, we ask that you report it to OpenStreetMap US Code of Conduct Committee via the process outlined in the [Process for Moderation](https://wiki.openstreetmap.org/wiki/Foundation/Local_Chapters/United_States/Code_of_Conduct_Committee/ModerationProcess). The CoC Committee will adjudicate reports submitted via this process, however, they will not actively monitor project communication channels.
+
+## Maintainer Responsibilities
+
+- The maintainer team agrees to abide by the CoC and to implement and be bound by moderation decisions made by the CoC Committee. This includes making a best effort to resolve disputes
+- Maintainers agree not to act in CoC cases in which a maintainer has a conflict of interest, such as in the case of a complaint made against the maintainer.
+
+## Applicability
+
+- This CoC applies to the OpenStreetMap Americana github repository. however, the CoC committee may consider a holistic examination of communications made via other channels when evaluating a complaint.
+- This CoC remains in force until or unless revoked by the maintainer team, the OSM US CoC Committee, or OSM US. Should the CoC be revoked, it remains applicable for all behavior which occurred prior to revocation.
+- The list of encouraged and untolerated behaviors are adopted as written at the time of the approval of this CoC. Should the standards of behavior in the OSM US CoC change, the maintainer team reserves the right to adopt the change, withdraw from the Process for Moderation, or neither.
+- The OSM US CoC Committee may at any time change the procedures outlined in the Process for Moderation and/or the Statement of Scope and Limitations as documented in the Committee's [online documentation](https://wiki.openstreetmap.org/wiki/Foundation/Local_Chapters/United_States/Code_of_Conduct_Committee/OSM_US_Code_of_Conduct). The project adopts these procedures as currently written, even if they differ from the procedures that were in force at the time of the project's adoption of the CoC.
+- If the locations of the official online documentation of the OSM US CoC and Process for Moderation change, the presently applicable documentation applies, and updates to these links may be made as an administrative change.
+
+## License
+
+Per a _(Date)_ decision of the OpenStreetMap US Board of Directors, OpenStreetMap US releases the text in this document from any claim of copyright, and grants permission for its inclusion in public domain-compatible works, with or without the inclusion of this statement.
```

Comment:
The OSM-US CoC looks to be licensed under either the [Creative Commons Attribution-ShareAlike 3.0](http://creativecommons.org/licenses/by-sa/3.0/) or [Creative Commons Attribution-ShareAlike 2.0](https://wiki.openstreetmap.org/wiki/Wiki_content_license).

As the `CODE_OF_CONDUCT.md` is not executable code that must be linked or compiled into any software package, my understanding is that it would be fine to include this file in the Americana repository under its existing Creative Commons Attribution-ShareAlike license without any public-domain waver or copyright release. My lay understanding of the applicable law is that such a license on this one file (with appropriate in-file links) would be sufficient to fulfill the attribution and share-alike demands of the license and would not force any of the rest of the code or other files also included in this repository to have the same license.

Additionally, the OSM-US is likely not in a position to grant such a release as the OSM-US CoC itself drew from upstream Creative Commons Attribution-ShareAlike licensed sources.

Reply:
Copyright is not just limited to executable code - it applies to any creative work.  We can't declare this repository to be CC0/public domain if there are portions of the repository that have copyright requirements (specifically - attribution).  If we (OSM US) aren't able to produce PD-compatible CoC text, I don't see how we (Americana project) can include the content in the repo.

Reply:
Per Slack discussion, I've updated the README with an exception disclaimer and we'll include the attribution piece here in this file.

Reply:
Correct that copyright isn't limited to executable code. That said, the "viral" nature of share-alike licenses (e.g. GPL) has been determined in litigation to stop at the compiling/linking/single-executable boundary and is separate from "collections of differently licensed applications".

In my mind, this file and the code are separate and simply "collected" together rather than being a single "work".